.circleci/config.yml

```diff
@@ -243,7 +246,10 @@ jobs:
             docker login -u $DOCKER_USER -p $DOCKER_PASS
             docker build --pull -t addons-frontend .
       - run: docker images
-      - run: docker run --rm addons-frontend node --version
+      - run:
+          name: Build the app and check bundlesize from container
+          command: |
+            docker run --rm addons-frontend yarn build-ci
```

Comment:
Same comment here.

In addition, we never check those CI results usually because GitHub does not show them by default. I think it'd be good to keep the step for debugging but bundlesize shouldn't stop a deploy.